test/app/commands.test.ts

```diff
@@ -325,6 +325,25 @@ describe('ExtensionManager: commands tests', function () {
       });
     });
 
+    context(`the toggle yaml preset command`, function () {
+      it(`toggles the yaml preset`, function () {
+        const toggleYamlPresetCommand =
+          // @ts-ignore
```

Comment:
IMO `@ts-ignore` should never be used. Use `@ts-expect-error` instead. That will fail if the comment is redundant.

Reply:
Agreed. I was just mimicking what we already had before to make this work. I think we should have another PR focusing on how we are using @ts-ignore in the extension and see if we can mitigate its usage.

Reply:
@ts-ignore was used because those functions are called via reflection and back in the day there was no other way to make ts lint silent.